### PR TITLE
Open stack page updates managed build 4215

### PR DESCRIPTION
**Pull request by inuwa**
## Done

Updated OpenStack page updates - Managed build to match [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#heading=h.uasr0ce2c7v3) 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack/managed](http://0.0.0.0:8001/openstack/managed)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#heading=h.uasr0ce2c7v3) and check to make sure that the page changes are reflected in the page

## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4215

### CODE REVIEW

**Comment by inuwa** [comment]
> @inuwa I'm slightly confused as you've made changes to **openstack/managed** but the copy doc refers to **openstack/features** - "Features of Canonical OpenStack on Ubuntu"
> 
> So I'm not sure what I'm having to review? 😄

😄Apologies the link was to the wrong file. I have updated the link to the copy doc now.

**Comment by kwm14** [comment]
## Design review
Looking good @inuwa just a couple of updates 👍 
- [ ] 'A selection of BootStack customers' logo Cloud should be moved below 'Managed private cloud, a perfect fit for multi-cloud environments' as per copy doc layout, you may have to alternate the background color dependent on what it follows
screenshot 2018-10-26 at 12 22 45
- [ ] I would add the quote from Tele2 with 'The smart approach to data center outsourcing' section as per the copy doc, feels disconnected be in it's own row. Just split the content `6-col` to fit alongside each-other 😄

**Comment by inuwa** [comment]
@kwm14 The reason we opted for the blockquote in a separate section was because it did not look right in-line. 
To make it look a bit okay (IMHO) 😀I reduced the citation font size.
image
This is what it looks like without the reduced font - 
image

**Comment by kwm14** [comment]
## Design review
@inuwa thanks for the updates looking good 👍 

I see what you mean when putting the block quote inline like that, also prefer the reduced citation font-size.

How about we move the block quote to the right and put inside `class="prefix-1 col-6"` to allow for padding between the content?

screenshot 2018-10-26 at 16 44 17

**Comment by inuwa** [comment]
@kwm14 it's done now, cheers.😅

**Comment by kwm14** [comment]
Thanks @inuwa looking really good. Apologies for all the small changes 😉 👍

File: templates/openstack/managed.html [hunk 1]
```diff
@@ -487,7 +487,7 @@ <h3 class="p-list-step__title">
 <section class="p-strip">
   <div class="row">
     <div class="col-12">
-      <h2>Shape it to your needs</h2>
+      <h2>Customisable Cloud</h2>
```
Comment (deadlight) [inline templates/openstack/managed.html]: Incorrect capitalisation: Should be "Customisable cloud" (sentence case)

**Comment by inuwa** [comment]
@deadlight Customisable Cloud to Customisable cloud. Cheers

**Pull request merged**